src/pre_process/m_checker.f90

```diff
@@ -2,6 +2,8 @@
 !!@file m_checker.f90
 !!@brief Contains module m_checker
 
+#define UNSUPPORTED_MESSAGE(f, s) "Unsupported combination of values of " + f + " and " + s + "Exiting ..."
```

Comment:
how is this used?

Reply:
I was testing something I must've forgotten to remove it.